src/tests/ftest/util/slurm_utils.py

```diff
@@ -139,25 +139,30 @@ def write_slurm_script(path, name, output, nodecount, cmds, uniq, sbatch=None):
 def run_slurm_script(script, logfile=None):
     """Run slurm script.
 
-    script(str) -- script file suitable to by run by slurm
-    logfile(str) -- add -o param and generate logfile
-    returns --the job ID, which is used as a handle for other functions
+    Args:
+        script (str): script file suitable to by run by slurm
+        logfile (str): add -o param and generate logfile
+
+    Returns:
+        str: the job ID, which is used as a handle for other functions
+
     """
+    job_id = None
+    cmd = ["sbatch"]
```

Comment:
Do we need a full path for the `sbatch` command?

Reply:
not needed.

Reply:
not needed.